api/src/main/resources/org/openmrs/api/db/hibernate/Encounter.hbm.xml

```diff
@@ -59,7 +59,7 @@
 			<column name="form_id" />
 		</many-to-one>
 		<!-- bi-directional one-to-many association to Order -->
-		<set name="orders" inverse="true">
+		<set name="orders" inverse="true" where="order_group_id is null">
```

Comment:
But as an API consumer, if i look up an encounter i expect Encounter.getOrders() to return all orders linked to it regardless of whether they belong to an order group or not, with this where in the mapping file, it means getOrders won't return orders that belong to order group even though they belong to this encounter